src/assets/images/nodejs.svg

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please use svg optimizer to reduce svg size
https://jakearchibald.github.io/svgomg/

Reply:
Done. But the size of nodejs.svg still big, maybe from of the complexity of their logo.
sass.svg can't optimize anymore since svgomg can only remove the viewBox attribute and that break their logo look.

Reply:
Cool!